<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>ToxicoProteomics Atlast of PFAS Toxicity</title>
    <meta charset="utf-8" />
    <script src="index_files/header-attrs-2.14/header-attrs.js"></script>
    <link rel="stylesheet" href="mod-metropolis.css" type="text/css" />
    <link rel="stylesheet" href="font.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">


count: false



&lt;!-- 
  colours: 
    oxford blue #0A214F
    White #FFFFFF
    Portland Orange #F46036
    Titatium Yellow #F0E100
    Persian Green #1B998B
--&gt;
 
*this slide intentionally left blank*

---
count: false
background-image: url(images/zebrafish-title.jpg)
background-position: center
background-size: cover
.inverse-title[

## ToxicoProteomics Atlas of Per- and Polyfluoroalkyl Substances 
### **David Hall**, Jiajun Han, Wen Gu, Diwen Yang, and Hui Peng. 
#### Dept. of Chemistry and School of the Environment&lt;br/&gt;University of Toronto
]

.inverse-bottomright[
S. Wilson&lt;/br&gt;Welcome Collection

]

???

- Confocal micrograph of zebrafish brain
- Green = GFP in neurons
- Red = axons &amp; neurofils
- Blue = synaptic vesicles 

---

background-image: url(images/74PFAS.png)
background-position: center
background-size: cover

## Much ado about PFAS

???

- 74 Compounds in USEPA's initial PFAS screening library 
- Mixture of Legacy and novel PFAS
- Varying chain length's
- varying head groups 
- **Enough of both to make comparisons...**
- How to do toxicogy testing given number of compounds? 
- image source: https://wellcomecollection.org/works/pftqz7hc
---


background-image: url(images/zebrafish_embryo.jpg)
background-position: center
background-size: cover

.inverse-bottomleft[Annia Cavanagh, Wellcome Collection ] 

???


- *Point out*: 
  - *EYES*
  - *Mouth* 
  - **Pericardium**, little sac (big sac is hatching gland) 
  
- Tox testing
  - *Embryo's are transparent*
  - vertebrates 
  - *Pericardial Edema*: fluid build-up around heart
  - *Spinal curvature*: tail supposed to detach from yoke-salk, and straighten out. 
  - *Tail Dysplasia*: malformations of tail fins.  
- False-coloured SEM of a zebrafish embryo.
- tropical freshwater fish originating from eastern Asia and is a member of the minnow family. 
- embryo as a disease model, an assay system for drug screening and is used in cancer research. 
- Suitable for biomedical rersearch: embryos develop quickly 
  - single cell in a fertilized egg to tiny fish in 24 hours 
  - vertebrates 
  

---

## OECD has adopted Fish Embryo Toxicity testing for acute fish toxicity measurements as it's logistically simpler and more reproducible 

&lt;img src="images/OECD_testing.png" width="900px" style="display: block; margin: auto;" /&gt;

.bottomright[
OECD &lt;/br&gt; 2022
]

???

- Fish embryos until the onset of independent feeding are considered as nonprotected life stages
- Easy to maintain in lab
- better interlab reproducibility 
- 


---

## Traditional fish embryo toxicity endpoints are poor predictors of &lt;/br&gt;weak acute and sublethal toxicities


&lt;img src="images/ehpDat.png" width="1000px" style="display: block; margin: auto;" /&gt;


???

- *20 embryos per exposure, 3 exposure replicates*
- **16**: N-Methylperfluorooctanesulfonamide
- **24**: N-Ethylperfluorooctanesulfonamide
- **34**: Perfluoro-3,6,9-trioxatridecanoic acid
- **36**: Methyl perfluorohexanoate
- **46**: Perfluorooctanesulfonic acid
- **60**: Perfluorooctanesulfonamide
- **74**: Perfluorooctanoic acid
- Single point testing is slow, and like finding a needle in a hay stack 
- Need a deeper picture to understand what's going
- Since proteins are movers and shakers, why not look at them. 

---

## Proteomics can measure tens of thousands of peptides to relatively quantity thousands of proteins

&lt;img src="images/proteomics.png" width="1000px" style="display: block; margin: auto;" /&gt;


???

- Matching peptide sequences to in silico predicted fragments to identify peptides.  

---
## Low- and high-conc PFAS exposures were performed in triplicate; labelling accounts for batch and matrix effects


&lt;img src="images/samplePrep.png" width="1000px" style="display: block; margin: auto;" /&gt;

???

- Low- and High- level exposures to better evaluate sub-lethal exposure effects 
- Dimethyl labelling enables relative quant and accounting for matrix effects. 

---
## Data Independent Acquisition(DIA) captures much more sample information compared to traditional Data-Dependant Acquisition (DDA)

&lt;img src="images/ludwig_etal.png" width="800px" style="display: block; margin: auto;" /&gt;

.bottomright[
Ludwig et al.,&lt;/br&gt; &lt;i&gt;Mol. Syst. Biol.&lt;/i&gt; (2018) &lt;/br&gt;
Cox et al., &lt;/br&gt;&lt;i&gt;Nat. Biotechnol.&lt;/i&gt; (2008) &lt;/br&gt;
Demichev et al., &lt;/br&gt;&lt;i&gt;Nat Methods&lt;/i&gt; (2020)
]

???

- **DDA**
  - One peptide sequenced / MS2 event
  - **Stochastic peak picking**, poor reproducibility. 
  - Finite amount of MS2 scan events, upper bound on DDA coverage 
  - straightforward peptide sequencing
  - Use **MaxQuant** as benchmark software
- **DIA**
  - multiple peptides / MS2 event
  - greater coverage 
  - More complex data + Spectral Library = greater software demands. 
  - **DIA-NN** uses neural networks for and interference for deep, robust, and reliable, proteome coverage 

---

.pull-left[

## **Recap**

### 1 -  Exposing zebrafish embryos to 74 PFAS


### 2 - Relatively quantify proteins &lt;/br&gt; through isotopic channels


### 3 - Latest DIA Software
]

???

- Onwards to how our data actuallly looks
- Begining with a comparison between the two software. 

---

## Randomly selected PFAS 31 exposure samples to assess DIA vs. DDA proteomics analysis





&lt;img src="images/31.png" width="300px" style="display: block; margin: auto;" /&gt;
PFAS 31: Heptafluorobutyramide





---
## DIA aquisition + DIA-NN software identifies more protein-groups per channel and per run compared to DDA + MaxQuant

.pull-left[

&lt;img src="images/proteinBars.png" width="500px" style="display: block; margin: auto;" /&gt;

]

--

.pull-right[

&lt;img src="images/DIAvsMQ.png" width="500px" style="display: block; margin: auto;" /&gt;

]

???

- UpsetPlot, similar to venn diagram but shows all intersections
- Lower number of proteins in these because only confidently ID proteins (one protein per group)
- 

---
## While DDA matches better within the same run, DIA-NN (plexDIA) performs better across runs

&lt;img src="images/jaccard_touchedup.png" width="1000px" style="display: block; margin: auto;" /&gt;

???

- Jaccard index shows the overlap between batches
- MQ better within the same run, by definition on how it works, but recall much less hits
- lack of overlap between runs will severely hamper downstream analysis b/c of stats 

---

## Long-chain perfluorinated PFAS 16 was one of the most bioaccumulative and the second most toxic tested PFAS

.pull-left[

&lt;img src="images/16.png" width="300px" style="display: block; margin: auto;" /&gt;
PFAS 16: N-Methylperfluoro&lt;/br&gt;octanesulfonamide

]

.pull-right[

&lt;img src="images/ehs_mortality.png" width="400px" style="display: block; margin: auto;" /&gt;

.bottomright[
Han et al.,&lt;/br&gt; &lt;i&gt;EHP&lt;/i&gt; (2021) 
]
]

???

- 2nd most toxic 
- Somewhat bio-accumulative 

---


## PFAS 16 significantly perturbes zebrafish embryo proteome,&lt;/br&gt; with &gt; 300 up- and &gt; 100 down-regulated proteins

.pull-left[

&lt;img src="images/ehs_mortality.png" width="400px" style="display: block; margin: auto;" /&gt;

.bottomleft[
Han et al.,&lt;/br&gt; &lt;i&gt;EHP&lt;/i&gt; (2021) 
]
]


.pull-right[

&lt;img src="images/pfas16_volcano.png" width="400px" style="display: block; margin: auto;" /&gt;



]

---

## Which zebrafish embryo proteins perturbed by PFAS 16 exposure potentially reveals mechanisms of toxicity

.pull-left[

&lt;img src="images/pfas16_volcano.png" width="400px" style="display: block; margin: auto;" /&gt;



]



.pull-right[

&lt;img src="images/pfas16_gobp_plot.png" width="400px" style="display: block; margin: auto;" /&gt;

]

???

- Only upregulated proteins for PFAS 16 
- makese sense given chemical insult 
- H2O2 and Eye Lense Development, possible oxidative stress and poor eye development
- These are two testable endpoints. 

---

background-image: url(images/analyzed_pfas.png)
background-position: center
background-size: cover

## Analyzed 22 PFAS exposures to date

???

- Blue boxes are compounds analyzed
- All samples are prepapred though, just waiting on instrument run-time. 

---
## Initial survey of global zebrafish embryo proteome perturbation begins to reveal trends

&lt;img src="images/mayHeatmap.png" width="1000px" style="display: block; margin: auto;" /&gt;

???

- clustering 
- elaborate on expanding scope to 74 pfas
- Interesting that 13 has a small batch of v. down regulated proteins, interesting because compound 13 is a small triflate.



---
## Conclusions &amp; Future Directions


.pull-left[

Thank you: 

**Conclusions**

- DIA aqusition &amp; DIA-NN enable a more complete survey of the zebrafish proteome. 
- PFAS exhibit structure specific proteome response in zebrafish embryo 

**Future Work**

- Finish zebrafish proteome analysis
- Chemometrics analysis of proteome response
- Verification experiments 


]

.pull-right[

&lt;img src="images/sig_barplot.png" width="400px" style="display: block; margin: auto;" /&gt;



]

---
## Ackwnowledgements

.pull-left[

Thank you: 

**Zebrafish Exposure&lt;/br&gt; and Sample prep**
- Jiajun Han
- Gu Wen
- Diwen Yang

**Zebrafish Rearing**

- Jiabo Liu
- Henry Kraus

**PFAS Library**

- Keith Houck
- US EPA (comments &amp; bad jokes are mine and not theirs)

]

---
## Questions? 

Email me (**davidross.hall@mail.utoronto.ca**) or Dr. Hui Peng (**hui.peng@utoronto.ca**)
  
&lt;img src="images/mayHeatmap.png" width="1000px" style="display: block; margin: auto;" /&gt;

  
---
## SI: We developped a Fluorous Solid Phase Enrichement strategy to enrich long-chain PFAS modified peptides

&lt;img src="images/fspe.png" width="1000px" style="display: block; margin: auto;" /&gt;

.bottomright[

Hall D. et al, &lt;/br&gt;
manuscript in preparation

]

---
## SI: 8:2 FTAC, and presumably other PFAS, can bind covalently, but not where expected

&lt;img src="images/ftac_mods.png" width="1000px" style="display: block; margin: auto;" /&gt;
.bottomright[

Hall D. et al, &lt;/br&gt;
manuscript in preparation

]
    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"ratio": "16:9"
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
// add `data-at-shortcutkeys` attribute to <body> to resolve conflicts with JAWS
// screen reader (see PR #262)
(function(d) {
  let res = {};
  d.querySelectorAll('.remark-help-content table tr').forEach(tr => {
    const t = tr.querySelector('td:nth-child(2)').innerText;
    tr.querySelectorAll('td:first-child .key').forEach(key => {
      const k = key.innerText;
      if (/^[a-z]$/.test(k)) res[k] = t;  // must be a single letter (key)
    });
  });
  d.body.setAttribute('data-at-shortcutkeys', JSON.stringify(res));
})(document);
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
